assets/js/payment-method-extensions/payment-methods/paypal/index.js

```diff
@@ -4,6 +4,7 @@
 import { registerPaymentMethod } from '@woocommerce/blocks-registry';
 import { __ } from '@wordpress/i18n';
 import { getSetting, WC_ASSET_URL } from '@woocommerce/settings';
+import { WC_BLOCKS_ASSET_URL } from '@woocommerce/block-settings';
```

Comment:
If this resulted in 1.34kb being added to the bundle build, that's a bit alarming and we should avoid it.

Reply:
It looks like `@woocommerce/block-settings` is not [defined as an external here](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/869741051e4d52d9f05ff63f5f0b62e6fea09f33/bin/webpack-helpers.js#L25-L29), which might also impact build sizes for all our bundles!